### PR TITLE
Mtmonir

### PR DESCRIPTION
**Pull request by mtmonir**
Please check my first commit

### CODE REVIEW

**Review by nicholasvvong — APPROVED** [review]
Good starting migration files from old project.

File: UserApi/Controllers/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+            if (user == null)
+            {
+                return StatusCode(404);
+            }
+            StatusCode(200);
+            return user;
+        }
+
+        [HttpDelete("delete/{userid}")]
```
Comment (markmoorerev) [inline UserApi/Controllers/UserController.cs]: needs an xml comment.

File: UserApi/Controllers/UserController.cs [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            {
+                return StatusCode(202);
+            }
+            else
+            {
+                return StatusCode(404);
+            }
+        }
+
+        [HttpPost("addadmin/{username}")]
```
Comment (markmoorerev) [inline UserApi/Controllers/UserController.cs]: xml comment

File: Logic/UserLogic.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    public class UserLogic : Interfaces.IUserLogic
+    {
+        private readonly RepoLogic _repo;
+        
+        public UserLogic(RepoLogic repo)
+        {
+            _repo = repo;
+        }
+
+        public async Task<bool> CreateUser(User user)
```
Comment (markmoorerev) [inline Logic/UserLogic.cs:24]: add xml comments to all these methods

File: Logic/Interfaces/IUserLogic.cs [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        /// <summary>
+        /// Returns a list of every User object.
+        /// </summary>
+        /// <returns></returns>
+        public Task<List<User>> GetUsers();
+
+        /// <summary>
+        /// Delete the user
+        /// Return true or false
+
```
Comment (markmoorerev) [inline Logic/Interfaces/IUserLogic.cs:44]: theres no summary closing tag

**Review by markmoorerev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**